src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,36 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+interface Props {
+  moviesList: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = ({ moviesList }) => (
+  <ul>
```

Comment:
You can delete ul and li tags and add you MovieCard in <div className="movies"> from App component. Then use map and in React fragment, which includes key={imdbId}, put MovieCard component. Because now your cards are placed in one column, that's why you need use class 'movies' from bulma library, i guess...)